fdbserver/DataDistributionTracker.actor.cpp

```diff
@@ -699,6 +730,15 @@ ACTOR Future<Void> shardEvaluator(DataDistributionTracker* self,
 	                   (bandwidthStatus == BandwidthStatusHigh && keys.begin < keyServersKeys.begin);
 	bool shouldMerge = stats.bytes < shardBounds.min.bytes && bandwidthStatus == BandwidthStatusLow;
 
+	std::vector<Key> manualSplitKeys;
+	if (shouldManualSplit->get() == true) {
+		// Force to split
+		shouldSplit = true;
+		shouldMerge = false;
+		manualSplitKeys.insert(manualSplitKeys.end(), manualSplitPoints->begin(), manualSplitPoints->end());
```

Comment:
Why are we doing a `merge` here, instead of `manualSplitKeys =  manualSplitPoints`? Are we assuming keys in `manualSplitKeys` are sorted?